docs/configuration.md

```diff
@@ -120,3 +120,110 @@ require("@babel/core").transform("code", {
 ```
 
 Check out the [babel-core documentation](core.md) to see more configuration options.
+
+## Print effective configs
+
+You can tell Babel to print effective configs on a given input path
+```sh
+# *nix or WSL
+BABEL_SHOW_CONFIG_FOR=./src/myComponent.jsx npm start
+```
+
+```powershell
+$env:BABEL_SHOW_CONFIG_FOR=.\src\myComponent.jsx; npm start
+```
+
+`BABEL_SHOW_CONFIG_FOR` accepts absolute and relative _file_ path. If it is a relative path, it will be resolved from [`cwd`](options.md#cwd).
```

Comment:
```suggestion
`BABEL_SHOW_CONFIG_FOR` accepts both absolute and relative _file_ paths. If it is a relative path, it will be resolved from [`cwd`](options.md#cwd).
```